app/views/events/_form.html.erb

```diff
@@ -0,0 +1,37 @@
+<%= form_with(model: event) do |form| %>
+  <% if event.errors.any? %>
+    <div id="error_explanation">
+      <h2><%= pluralize(event.errors.count, "error") %> prohibited this event from being saved:</h2>
+
+      <ul>
+        <% event.errors.each do |error| %>
+          <li><%= error.full_message %></li>
+        <% end %>
+      </ul>
+    </div>
+  <% end %>
```

Comment:
Move this logic to a helper. There should be absolutely no logic in the view. Views are used to render data that are passed to them by the controller, and not for generating it.